go.mod

```diff
@@ -1,21 +1,31 @@
 module rke-tools
 
-go 1.16
+go 1.20
```

Comment:
Build container is go 1.19 but module file version is on 1.20

Reply:
Reverted this back to `1.19`.